client/extensions/woocommerce/app/store-stats/store-stats-referrer-widget-base/index.js

```diff
@@ -0,0 +1,179 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Table from 'woocommerce/components/table';
+import TableRow from 'woocommerce/components/table/table-row';
+import Card from 'components/card';
+import ErrorPanel from 'my-sites/stats/stats-error';
+import { getWidgetPath } from 'woocommerce/app/store-stats/utils';
+import Pagination from 'components/pagination';
+import { getStoreReferrersByDate } from 'state/selectors';
```

Comment:
Strangely, once rebased to master this is throwing an error. I think it should be:
`import getStoreReferrersByDate from 'state/selectors/get-store-referrers-by-date';`

Reply:
Could potentially be related to this p4TIVU-8Vl-p2